lib/public/views/Runs/RunPerPeriod/RunsPerLhcPeriodOverviewPage.js

```diff
@@ -60,12 +65,20 @@ export const RunsPerLhcPeriodOverviewPage = ({ runs: { perLhcPeriodOverviewModel
     };
 
     return h('', [
-        h('.flex-row.justify-between.items-center', [
+        h('.flex-row.justify-between.items-center.g2', [
```

Comment:
Is there a need for adding `g2`? I would argue due to the usage of `justify-between`, you do not need either `items-center` or `g2`

Reply:
`g2` removed, without `items-center` 'Export runs' goes into bad position